### PR TITLE
Issue/jperf 567 jsw milestone releases

### PR DESCRIPTION
**Pull request by dagguh**
https://ecosystem.atlassian.net/browse/JPERF-567

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,15 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.14.0...master
 
+### Added
+- Expose `AwsCli`. Provide workarounds for [JPERF-567] and the like.
```
Comment (wyrzyk) [inline CHANGELOG.md]: 👍

**Comment by dagguh** [comment]
https://ecosystem.atlassian.net/browse/JPERF-567

File: CHANGELOG.md (hunk 1 above)
Comment (wyrzyk) [reply CHANGELOG.md]: I made a local workaround in our internal tests to use AwsCli, then I forgot to promote it to API. Thanks for the change. I will finally clean this workaround.

Comment (dagguh) [reply CHANGELOG.md]: No problem 🙂 
Fun fact: when you bump to `aws-infrastructure:2.15.0` (released after we merge this), the workaround will stop working at compile time. But the proper fix will be available within the same bump, so you should be ok

**Pull request merged**